testing/php7-brotli/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=php7-brotli
+_pkgreal=brotli
+pkgver=0.5.2
+pkgrel=0
+pkgdesc="Brotli Extension for PHP" 
+url="https://github.com/kjdev/php-ext-brotli"
+arch="all"
+license="MIT"
+options="!check" # As of 0.5.2 2/3 test fails
```

Comment:
Whould be great to get why thay fails, probably it needs more php extensions

Reply:
```
FAIL Test compatibility [tests/compatibility.phpt] 
PASS Test phpinfo() displays brotli info [tests/info.phpt] 
FAIL Test compatibility [tests/roundtrip.phpt] 
```

Reply:
The problem with tests is because missing test data https://github.com/google/brotli/tree/master/tests
https://pkgs.alpinelinux.org/contents?branch=edge&name=brotli-dev&arch=x86_64&repo=testing shows it was not packaged

Reply:
@andypost Thanks for the investigation 👍  I have updated the comment.